arch/arm/boot/dts/overlays/i2c-sensor-overlay.dts

```diff
@@ -231,11 +231,25 @@
 		};
 	};
 
+	fragment@15 {
+		target = <&i2c_arm>;
+		__dormant__ {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			status = "okay";
+
+			sps30: sps30@69 {
+				compatible = "sensirion,sps30";
+				reg = <0x69>;
+				status = "okay";
+			};
+		};
+	};
 
 	__overrides__ {
 		addr =  <&bme280>,"reg:0", <&bmp280>,"reg:0", <&tmp102>,"reg:0",
 			<&lm75>,"reg:0", <&hdc100x>,"reg:0", <&sht3x>,"reg:0",
-			<&ds1621>,"reg:0", <&bme680>,"reg:0";
+			<&ds1621>,"reg:0", <&bme680>,"reg:0", <&sps30>,"reg:0";
```

Comment:
If addresses are fixed, you don't need this change.

Reply:
Ok, fixed.